src/main/java/seedu/address/logic/commands/UnpaidCommand.java

```diff
@@ -58,6 +57,7 @@ public CommandResult execute(Model model) throws CommandException {
 
         Person originalPerson = lastShownList.get(index.getZeroBased());
         Person unpaidPerson = createUnpaidPerson(originalPerson, unpaidPersonDescriptor);
+        System.out.println(unpaidPerson);
```

Comment:
Believe this line is for testing purposes and should be removed.

Reply:
Yes this is not supposed to be there 👍